Bottom: a49b929640211788b76e3c0e0126d2e0415f2977
Top:    d3686af7b62bcd9abc68d359ef5af3e2fb1eb416
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 18:50:50 +0100

Refresh of hid-gtk-fixup-opengl-includes-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 0bbd621..21edb5d 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -38,8 +38,17 @@
  *   http://www.opengl.org/registry/ABI/
  */
 #define GL_GLEXT_PROTOTYPES 1
-#include <GL/gl.h>
-#include <GL/glu.h>
+#ifdef HAVE_OPENGL_GL_H
+#   include <OpenGL/gl.h>
+#else
+#   include <GL/gl.h>
+#endif
+
+#ifdef HAVE_OPENGL_GLU_H
+#   include <OpenGL/glu.h>
+#else
+#   include <GL/glu.h>
+#endif
 
 #include "action.h"
 #include "crosshair.h"
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1f51fc1..15c9468 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -17,8 +17,14 @@
  * in order to get prototypes:
  *   http://www.opengl.org/registry/ABI/
  */
+
 #define GL_GLEXT_PROTOTYPES 1
-#include <GL/gl.h>
+#ifdef HAVE_OPENGL_GL_H
+#   include <OpenGL/gl.h>
+#else
+#   include <GL/gl.h>
+#endif
+
 #include <gtk/gtkgl.h>
 #include "hid/common/hidgl.h"
 #include "hid/common/draw_helpers.h"
